lib/src/load_balancing.rs

```diff
@@ -28,6 +28,7 @@ impl LoadBalancingAlgorithm for RoundRobinAlgorithm {
 
 impl RoundRobinAlgorithm {
 
+  #[allow(dead_code)]
```

Comment:
please remove that attribute. A dead code warning usually indicates that an API has an issue (I'm planning to rework the load balancing part), so just allowing it instead of fixing the issue is often a bad idea